msg/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(msg_files
 	heater_status.msg
 	home_position.msg
 	hover_thrust_estimate.msg
+	ice_status.msg
```

Comment:
Nitpicky, but is "ice" a well known abbreviation? I think I'd prefer just spelling it out so at a glance the the log is obvious.

Reply:
fixed